src/monitoring_service/cli.py

```diff
@@ -57,9 +65,19 @@ def main(  # pylint: disable=too-many-arguments
     confirmations: BlockNumber,
     min_reward: int,
     debug_shell: bool,
+    accept_disclaimer: bool,
 ) -> int:
     """ The Monitoring service for the Raiden Network. """
     log.info("Starting Raiden Monitoring Service")
+    click.secho(
```

Comment:
Why doesn't this get put on one line?

Reply:
I do not understand your comment, can we have a short call about that?

Reply:
I think he just talks about the formatting. If you run black on that file, this line and the following ones should be merged into a single line.

Reply:
ah I see, but my black tells me otherwise :)
at least I did run black several times on that file and it did not change. I can change by hand though.

Reply:
done